modules/default/currentweather/currentweather.js

```diff
@@ -587,6 +587,6 @@ Module.register("currentweather", {
 	 */
 	roundValue: function (temperature) {
```

Comment:
This function is used in 3 places. Could it be moved into some kind of utility class?

Reply:
I....don't know? I didn't see any obvious place to placfe functions that are shared across modules.

Reply:
Then nevermind, just some random thought of mine....